release

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -eo pipefail
+set -x
+curl https://bootstrap.pypa.io/get-pip.py -o /tmp/get-pip.py
+python /tmp/get-pip.py
+
+rm -rf dist dash.egg-info
+pip install -e .[testing,dev]
+python setup.py sdist
+
+cd dash-renderer
+rm -rf dist node_modules dash_renderer*
+# build renderer bundles, this will build all bundles from source code
+# the only true source of npm version is defined in package.json
+npm run build  # or `renderer build`
+# install dash-renderer for development
+pip install -e .
+# you should see both dash and dash-renderer are pointed to local source repos
+pip list | grep dash
+python setup.py sdist
```

Comment:
Can you explain the goal here?

Reply:
My fault. I wrote this script so I could run the release process in docker so I could build the package locally.